Packs/Shodan/Integrations/Shodan_v2/Shodan_v2.py

```diff
@@ -20,12 +20,7 @@
 # Should we use SSL
 USE_SSL = not demisto.params().get('insecure', False)
 
-# Remove proxy if not set to true in params
-if not demisto.params().get('proxy'):
-    os.environ.pop('HTTP_PROXY', None)
-    os.environ.pop('HTTPS_PROXY', None)
-    os.environ.pop('http_proxy', None)
-    os.environ.pop('https_proxy', None)
+handle_proxy()
```

Comment:
Are we sure that `requests.request` respects the proxy environment variables and that you don't need to explicitly pass the proxies dictionary returned by `handle_proxy` to the request created in `http_request`?

Reply:
yes, look at https://requests.readthedocs.io/en/master/user/advanced/#proxies

Reply:
I know. I read that prior to posting. But it seems odd that in our `BaseClient` we have to alter an instance variable to get the proxies working correctly but here we don't because `requests.get` etc. use a session under the hood... so it would seem to me that either our `BaseClient` or the integrations where we do `requests.(request/get/post/etc)` are doing something wrong or unnecessary.